routes.js

```diff
@@ -182,6 +184,25 @@ module.exports = function ( master ) {
 					return reply( 'Good' );
 				} );
 			}
+		},
+		// Create File associated with browserstack id
+		{
+			'method'  : 'POST',
+			'path'    : '/test-cases/{testCaseId}/logfile',
+			'handler' : function ( request, reply ) {
+
+				var testCaseId  = request.params.testCaseId;
+				var writeStream = fs.createWriteStream( logPath + '/' + testCaseId, { 'flags' : 'w' } );
+				writeStream.on( 'error', function ( error ) {
+					reply( 'Failed to create file' ).code( 500 );
+				} );
+				writeStream.on( 'close', function () {
+					reply( 'Success' );
+				} );
+				writeStream.write( '' );
```

Comment:
Instead of using createWriteStream what if just ```open``` ? 
https://nodejs.org/api/fs.html#fs_fs_open_path_flags_mode_callback